proto/rill/runtime/v1/resources.proto

```diff
@@ -243,6 +243,27 @@ message MetricsViewSpec {
     // The data type of the measure. Only populated in ValidSpec.
     Type data_type = 15;
   }
+  // Annotations that can be applied to measures
+  message Annotation {
```

Comment:
Would be good to explicitly document the expected/supported underlying columns for annotations here

Reply:
This is more like the annotation definition. Is this the right place for that description?

Reply:
For end-user descriptions, the YAML docs are a better place, but for (internal) info about how the resource works/validates/should be consumed, the protos are probably the best place.

Reply:
Sounds good, lemme update.